python-sdk/src/astro/sql/__init__.py

```diff
@@ -4,14 +4,11 @@
 
 from astro.sql.operators.append import AppendOperator, append
 from astro.sql.operators.cleanup import CleanupOperator, cleanup
-from astro.sql.operators.data_validations.ColumnCheckOperator import (  # skipcq: PY-W2000
+from astro.sql.operators.data_validations.column_check import (  # skipcq: PY-W2000
```

Comment:
you can remove `# skipcq: PY-W2000 ` by adding  `ColumnCheckOperator` and `column_check` in `__all__`  list